Regression/WarpX-tests.ini

```diff
@@ -1026,8 +1027,9 @@ analysisOutputImage = Langmuir_multi_2d_MR.png
 
 [Langmuir_multi_2d_MR_anisotropic]
 buildDir = .
-inputFile = Examples/Tests/langmuir/inputs_2d
-runtime_params = algo.maxwell_solver = ckc  warpx.use_filter = 1  amr.max_level = 1 amr.ref_ratio_vect = 4 2 warpx.fine_tag_lo = -10.e-6 -10.e-6  warpx.fine_tag_hi = 10.e-6 10.e-6  diag1.electrons.variables = x z w ux uy uz  diag1.positrons.variables = x z w ux uy uz
+inputFile = Examples/Tests/langmuir/inputs_2d_base
+aux1File  = Examples/Tests/langmuir/inputs_2d_test_Langmuir_multi_2d_MR_anisotropic
+runtime_params = FILE=inputs_2d_test_Langmuir_multi_2d_MR_anisotropic
```

Comment:
@ax3l @RemiLehe 

By the way, I think this type of input files organization (which came up here just as a way to guarantee backwards compatibility while setting up CTest) is quite good regardless of CTest itself.

At least all input parameters can be found in files from the same directory (e.g., Examples/Tests/langmuir/), rather than combining files from two directories (e.g., Examples/Tests/langmuir/ and Regression/).

I just wanted to point this out so that it doesn't get lost within the other (main) changes in this PR.

Reply:
Discussed in person:

We will use `FILE=` in the concrete input file we register with a test to reference a base parameter set. That way, we can avoid all CLI extra parameters and extra options.